### PR TITLE
updated deprecated syntax

### LINKED ISSUE #29: asgiref version conflict with Django 4.0.7+
Reproduce:
- Update Django to 4.0.7+
- It requires `asgiref<4` and `asgiref>=3.4.1`
- however, `asgiref` past 3.2.1 makes django throw an ipinfo exception: `'IPinfo' object has no attribute '_is_coroutine'`

### PR DESCRIPTION
**Pull request by rm-Umar**
fixes #29

### CODE REVIEW

File: ipinfo_django/middleware.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class IPinfo(MiddlewareMixin):
```
Comment (UmanShahzad) [inline ipinfo_django/middleware.py:14]: The import remains.

**Review by UmanShahzad — COMMENTED** [review]
Can you link some docs/etc that show this is the right move?

Also note that this is a breaking change and we need to update versions/etc to a new major version.

**Comment by rm-Umar** [comment]
> Can you link some docs/etc that show this is the right move?
> 
> Also note that this is a breaking change and we need to update versions/etc to a new major version.

A new middleware syntax was introduced in Django 1.10 and the old syntax was deprecated. It was also causing issues on Django version 4 on my side, so it required changing to newer syntax. Also, we are supporting Django 3 and 4 so we should support the latest syntax. 
https://docs.djangoproject.com/en/1.10/topics/http/middleware/

**Review by UmanShahzad — APPROVED** [review]
Alright, is this really the reason for the issues faced by #29 ?

**Comment by rm-Umar** [comment]
Yes, I was also getting the same error `AttributeError: 'IPinfo' object has no attribute '_is_coroutine'` on Django 4.1

> Alright, is this really the reason for the issues faced by #29 ?

**Comment by UmanShahzad** [comment]
Released https://pypi.org/project/ipinfo-django/2.0.0/

**Pull request merged**